draft-ietf-rtcweb-jsep.xml

```diff
@@ -857,23 +905,15 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
         <section title="createOffer" anchor="sec.createoffer">
           <t>The createOffer method generates a blob of SDP that contains a
           <xref target="RFC3264"></xref> offer with the supported
-          configurations for the session, including descriptions of the local
-          MediaStreams attached to this PeerConnection, the codec/RTP/RTCP
+          configurations for the session, including descriptions of the
+          RtpTransceivers added to this PeerConnection, the codec/RTP/RTCP
```

Comment:
This doesn't seem right. It's not the Transceivers that are being described by the associated media